src/calendar/index.js

```diff
@@ -0,0 +1,69 @@
+const moment = require('moment')
+
+const constants = require('./constants')
+
+const getTimezoneOffset = (offerPackageTimezoneOffset, offerType, requestTimezoneOffset) => {
+  if (!requestTimezoneOffset && constants.OFFERS[offerType].useTimezoneOffset) {
+    return parseInt(offerPackageTimezoneOffset || 0)
+  }
+
+  return requestTimezoneOffset
+}
+
+const getMonthsToRequest = (timezoneOffset, maxDate) => {
+  const now = moment().utcOffset(timezoneOffset)
+  const diff = moment.duration(moment(maxDate).diff(now))
+  return diff.months() + (12 * diff.years()) + 2 // Add 1 month for current month, add 1 month for end month
```

Comment:
How about?
```suggestion
return moment(maxDate).diff(now, 'months') + 2
```

Another question do we always need to add 1 month for the end month?

Reply:
Good question I just to move function from calendar

Reply:
Updated function